scipy/interpolate/interpolate.py

```diff
@@ -1650,9 +1649,10 @@ def from_derivatives(cls, xi, yi, orders=None, extrapolate=None):
                 n2 = min(n - n1, len(y2))
                 n1 = min(n - n2, len(y2))
                 if n1+n2 != n:
-                    raise ValueError("Point %g has %d derivatives, point %g"
-                            " has %d derivatives, but order %d requested" %
-                            (xi[i], len(y1), xi[i+1], len(y2), orders[i]))
+                    raise ValueError(
+                        "Point %g has %d derivatives, point %g"
+                        " has %d derivatives, but order %d requested" % (
+                            xi[i], len(y1), xi[i+1], len(y2), orders[i]))
```

Comment:
Nit: better separate assembling the string into a variable (mesg = "..."), then raise ValueError(mesg)